packages/components/src/border-control/border-control-dropdown/component.tsx

```diff
@@ -166,7 +166,7 @@ const BorderControlDropdown = (
 			onClick={ onToggle }
 			variant="tertiary"
 			aria-label={ toggleAriaLabel }
-			position={ dropdownPosition }
+			tooltipPosition={ dropdownPosition }
```

Comment:
`<Button>` has no `position` prop, so maybe this should be `tooltipPosition`. 

`tooltipPosition` and `dropdownPosition` have the same type.